src/js/utils/__tests__/ServiceUtil-test.js

```diff
@@ -324,16 +336,93 @@ describe('ServiceUtil', function () {
             .toEqual(1234);
         });
 
-        it('should not add a hostPort when loadBalanced is off', function () {
-          let service = ServiceUtil.createServiceFromFormModel({
-            containerSettings: {image: 'redis'},
+        it('should not remove host port', function () {
+          const definition = {
+            container: {
+              docker: {
+                image: 'docker/image',
+                portMappings: [
+                  {
+                    containerPort: 514,
+                    servicePort: 10000,
+                    hostPort: 5514
+                  }
+                ],
+                network: 'BRIDGE'
+              }
+            }
+          };
+
+          const model = {
+            containerSettings:{
+              forcePullImage: true,
+              image: 'docker/image',
+              parameters: null,
+              privileged: undefined
+            },
             networking: {
               networkType: 'bridge',
-              ports: [{lbPort: 1234}]
+              ports: [{
+                expose: true,
+                lbPort: 514,
```

Comment:
`lbPort` is converted either to `port`, `containerPort` or `servicePort` depending on your network and container settings.